File: Server/updateNotifications.php
<?php
  require "globalFunctions.php";
  $pdo = new PDO("pgsql:host=".HOST.";port=".PORT.";dbname=".DATABASE.";user=".USERNAME.";password=".PASSWORD);

  $myId = $_POST["myId"];
  $token = $_POST["token"];
  $notifyLikes = $_POST["notifyLikes"];
  $notifyComments = $_POST["notifyComments"];
  $notifyMessages = $_POST["notifyMessages"];

	$isValid =
		isAuthenticated($myId, $token) &&
		isValidBool($notifyLikes) &&
		isValidBool($notifyComments) &&
		isValidBool($notifyMessages);
	if (!$isValid) {
  	$pdo = null;
		die("Invalid");
	}

  $query = "update account set notify_likes = ?, notify_comments = ?, notify_messages = ? where user_id = ?;";
  $stmt = $pdo->prepare($query);
  $stmt->execute(array($notifyLikes, $notifyComments, $notifyMessages, $myId));

	$pdo = null;
?>

File: Server/updateNotifications.go
package main

import (
	"context"
	"fmt"
	"net/http"
)

func updateNotifications(w http.ResponseWriter, r *http.Request) {
	r.ParseForm()
	token := r.Form.Get("token")
	notifyLikes := r.Form.Get("notifyLikes")
	notifyComments := r.Form.Get("notifyComments")
	notifyMessages := r.Form.Get("notifyMessages")
	myId := getMyId(token)

	isValid :=
		isAuthenticated(myId) &&
			isValidBool(notifyLikes) &&
			isValidBool(notifyComments) &&
			isValidBool(notifyMessages)
	if !isValid {
		fmt.Fprintln(w, "Invalid")
		return
	}

	_, err := pool.Exec(context.Background(), "update account set notify_likes = $1, notify_comments = $2, notify_messages = $3 where user_id = $4;", notifyLikes, notifyComments, notifyMessages, myId)
	handleError(err)
}
